src/main/java/com/google/api/codegen/util/CommonRenderingUtil.java

```diff
@@ -24,13 +23,62 @@
  */
 public class CommonRenderingUtil {
 
+  /**
+   * Returns the input text split on newlines.
+   */
   public static List<String> getDocLines(String text) {
-    // TODO: convert markdown to language-specific doc format
+    // TODO: Convert markdown to language-specific doc format.
     // https://github.com/googleapis/toolkit/issues/331
     List<String> result = new ArrayList<>();
     for (String line : Splitter.on(String.format("%n")).split(text)) {
       result.add(line);
     }
     return result;
   }
+
+  /**
+   * Returns the input text split on newlines and maxWidth.
+   */
+  public static List<String> getDocLines(String text, int maxWidth) {
+    maxWidth = maxWidth - 1;
```

Comment:
I copied this code from context common I think...
I suspect it's a simple trick to make sure lines break on char 79 instead of char 80.

I'll verify and comment.